cartography/cli.py

```diff
@@ -124,6 +124,14 @@ def _build_parser(self):
                 'jobs are executed.'
             ),
         )
+        parser.add_argument(
```

Comment:
It's needed in case of Okta, there's no way I could find to get that information using the API Token.

Reply:
@ecdavis @achantavy what is the recommended way to pass data to intel modules beyond credentials? Should we all use env variable, cli param or case by case?

Reply:
I think that CLI parameters should control settings that are in common for _all_ syncs.  Granted, `--aws-sync-all-profiles` is a bit of an exception, but I think that it would be unwieldy to expose params for each intel module through the CLI.  For this reason I think that it would be best to use environment variables.  

What do you think @ecdavis?

Reply:
@ecdavis what's your take?